src/integration_tests/telemetry_async.cpp

```diff
@@ -244,6 +252,14 @@ void print_fixedwing_metrics(Telemetry::FixedwingMetrics fixedwing_metrics)
     _received_fixedwing_metrics = true;
 }
 
+void print_ground_truth(Telemetry::GroundTruth ground_truth)
+{
+    std::cout << "Latitude: " << ground_truth.latitude_deg << " deg, "
+              << "Longitude: " << ground_truth.longitude_deg << " deg, "
+              << "Altitude: " << ground_truth.absolute_altitude_m << " m" << std::endl;
```

Comment:
You could actually just do `std::cout << ground_truth << std::endl;` since you implemented it.